ivy/functional/frontends/torch/tensor.py

```diff
@@ -525,6 +525,11 @@ def flatten(self, start_dim, end_dim):
     def cumsum(self, dim, dtype):
         return torch_frontend.cumsum(self._ivy_array, dim, dtype=dtype)
 
+    @with_unsupported_dtypes({"1.11.0 and below": ("float16",)}, "torch")
+    def cumsum_(self, dim, dtype=None):
```

Comment:
`dtype` has to be a keyword argument, in order for the test to detect this automatically, you should add `*,` between `dim` and `dtype` (i.e. `def cumsum_(self, dim, *, dtype=None):`)